src/app/components/lineage-graph/lineage-graph-heigth.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from "@angular/core";
+
+@Injectable({
+    providedIn: "root",
+})
+export class LineageGraphHeigthService {
```

Comment:
Let's try to be higher level:
- name the service as `WidgetHeightService`
- move it somewhere higher level, maybe into shared services or something like this
- store a map with id/type name as a key, height as a value
- reduce number of methods for get/set, basically 2 with key argument will be enough